control-plane/stor-port/src/types/v0/transport/volume.rs

```diff
@@ -7,7 +7,6 @@ use crate::{
 use serde::{Deserialize, Serialize};
 use std::{collections::HashMap, convert::TryFrom, fmt::Debug};
 
-rpc_impl_string_uuid!(SnapshotId, "UUID of a snapshot");
```

Comment:
Is this not needed? Where did we define this?

Reply:
this is defined by my PR v0/transport/snapshot.rs that got merged